yarn-project/circuits.js/src/contract/events/contract_class_registered_event.test.ts

```diff
@@ -12,7 +12,7 @@ describe('ContractClassRegisteredEvent', () => {
     expect(event.artifactHash.toString()).toEqual('0x072dce903b1a299d6820eeed695480fe9ec46658b1101885816aed6dd86037f0');
     expect(event.packedPublicBytecode.length).toEqual(27090);
     expect(computePublicBytecodeCommitment(event.packedPublicBytecode).toString()).toEqual(
-      '0x1d5c54998c08cee8ad4a8af5740f2e844fe6db3a5bb4b6382a48b2daeabeee3f',
+      '0x0000000000000000000000000000000000000000000000000000000000000005',
```

Comment:
Can you add a ref to 5860 as you have above, got confused when i looked a second ago